doc/user/content/ops/network-security/ssh-tunnel.md

```diff
@@ -51,6 +51,49 @@ CREATE CONNECTION ssh_connection TO SSH TUNNEL (
     echo "ssh-ed25519 AAAA...76RH materialize" >> ~/.ssh/authorized_keys
     ```
 
+1. Configure your internal firewall to allow the SSH bastion host to connect to your Kafka cluster or PostgreSQL instance.
+
+    If you are using a cloud provider like AWS or GCP, update the security group or firewall rules for your PostgreSQL instance or Kafka brokers.
+
+    You'll need to allow incoming traffic from the IP address of the SSH bastion host on the necessary ports.
+
+    For example, use port `5432` for PostgreSQL and ports `9092`, `9094`, and `9096` for Kafka.
+
+    After that, test the connection from the bastion host to the Kafka cluster or PostgreSQL instance.
+
+    ```bash
```

Comment:
I think it makes sense to be slightly more prescriptive. Something like, if the command hands, the security group is likely not configured correctly. But if it shows this output, you can move to the next step.